saml/models/core/response.go

```diff
@@ -1,111 +1,9 @@
 package core
 
 import (
-	"encoding/xml"
-	"time"
+	"github.com/russellhaering/gosaml2/types"
 )
 
-type Response struct {
```

Comment:
I was thinking to leave the types here, but It makes sense to remove the types for now as they are incomplete and can lead to confusion.